broker/amqp.go

```diff
@@ -55,6 +64,27 @@ func (a *AMQP) Connect(url string) error {
 		false,
 		nil,
 	)
+
+	// setup RPC callback queue
+	rpc, err := a.channel.QueueDeclare(
```

Comment:
A couple suggestions to keep this roughly similar to existing implementations:

- Use default empty string name
- Leverage `autoAck` since the published result of an RPC call can never be retried if the connection is dropped anyway
- Make this queue exclusive to the connection to prevent any overlapping

```
channel.QueueDeclare("", false, true, true, false, nil)
```